docs/tracing_reference.rst

```diff
@@ -29,10 +29,10 @@ the request flow.
    .. attribute:: on_request_start
 
       Property that gives access to the signals that will be executed when a
-      request starts, based on the :class:`~signals.Signal` implementation.
+      request starts, based on the :class:`aiohttp.signals.Signal` implementation.
 
       The coroutines listening will receive as a param the ``session``,
-      ``trace_config_ctx``, ``method``, ``url`` and ``headers``.
+      ``trace_config_ctx`` and :class:`aiohttp.TraceRequestStartParams` params.
```

Comment:
Minor question.
Maybe ```The signal handler signature is ``async def on_request_start(session, context, params): ...`` where ``params`` is :class:`aiohttp.TraceRequestStartParams` instance``` for better readability?

The same for all other tracing signals.

Reply:
Done!